src/Sylius/Behat/Element/Admin/Product/AttributesFormElement.php

```diff
@@ -143,12 +141,14 @@ public function getNumberOfAttributes(): int
     protected function getDefinedElements(): array
     {
         return [
+            'product_attribute_value' => '[data-test-product-attribute-name="%attribute_name%"][data-test-locale-code="%locale_code%"]',
+            'form' => '[data-live-name-value="sylius_admin:product:form"]',
```

Comment:
Wrong sorting

Reply:
It will be covered in another PR with ProductAttribute related changes